### ISSUE TITLE
Scala-cli doesn't output`-coverage-out` products.

### ISSUE THREAD

**Issue by pikinier20** [VirtusLab/scala-cli#1024]
**Version(s)**
checked on 0.1.2 and 0.1.5

**Describe the bug**

After adding:
```
//> using options "-coverage-out:scoverage.coverage"
```
and running:
```
scala-cli compile -S 3.nightly .
```
scala-cli passes the option to compiler but the `scoverage.coverage` directory is not present neither in cwd nor in the `.scala-build` directory
**To Reproduce**
GoodCoverage.scala
```
//> using options "-coverage-out:scoverage.coverage"
object GoodCoverage {

  def sum(num1: Int, num2: Int) = {
    if (0 == num1) num2 else if (0 == num2) num1 else num1 + num2
  }

}
```

```
scala-cli compile -S 3.nightly .
```

**Expected behaviour**
There should be a directory `scoverage.coverage` containing output from the compiler which is essential to check code coverage with e.g. scoverage.

**Comment by lwronski** [comment]
Hi @pikinier20 , Thanks for reporting,
it seems that is also bloop issue as in #1023 

There is exists a workaround and you have to use plain `scalac` without `bloop`. To disable using `bloop` you should `use --server=false` flag:
```
$ scala-cli compile -S 3.nightly . --server=false
$ ls
Hello.scala		scoverage.coverage
```

**Comment by lwronski** [comment]
Hi @pikinier20,
Can you try to pass the absolute path to `-coverage-out` option? it should work when you pass the absolute path.

```
$ pwd
/tmp/scala-demo
$ cat Hello.scala 
//> using options "-coverage-out:/tmp/scala-demo/scoverage.coverage"
object GoodCoverage {

  def sum(num1: Int, num2: Int) = {
    if (0 == num1) num2 else if (0 == num2) num1 else num1 + num2
  }

}
$ scala-cli compile -S 3.nightly .
Compiling project (Scala 3.2.0-RC1-bin-20220606-cec9aa3-NIGHTLY, JVM)
Compiled project (Scala 3.2.0-RC1-bin-20220606-cec9aa3-NIGHTLY, JVM)
$ ls
Hello.scala		scoverage.coverage
```

**Comment by lwronski** [comment]
It should be fixed by #1080
